api/controllers/MerchStoreController.ts

```diff
@@ -164,10 +170,11 @@ export class MerchStoreController {
   }
 
   @Get('/order')
```

Comment:
This would make more sense for this particular route, but wouldn't we want all routes under the same resource to have the same prefix? i.e. any operation on the OrderModel resource is currently under `/store/order`, but with this change we'd have `/store/orders` and /store/order`. 

With events we have `/event/past` and `/event/future` as singular even though they return many events, so do we want to keep the same convention here? I don't have a strong preference here

Reply:
I like `/orders` plural because it's obvious the route is for getting several orders, but your call.